### PR TITLE
Dev build should not update _versions.yml file

### PR DESCRIPTION
**Pull request by mishig25**
Dev build should not update _versions.yml file

Reusing this bool/flag https://github.com/huggingface/doc-builder/blob/0e9fffb822ef46edc5aa10c0f44baa6193e41732/src/doc_builder/commands/build.py#L29

We can add other agument like `if arg PR_number exists` wdyt

### CODE REVIEW

File: src/doc_builder/commands/build.py [hunk 1]
```diff
@@ -46,8 +46,9 @@ def build_command(args):
         notebook_dir=args.notebook_dir,
     )
 
+    # dev build should not uodate _versions.yml
     package_doc_path = os.path.join(args.build_dir, args.library_name)
-    if os.path.isfile(os.path.join(package_doc_path, "_versions.yml")):
+    if "dev" not in version and os.path.isfile(os.path.join(package_doc_path, "_versions.yml")):
```
Comment (sgugger) [inline src/doc_builder/commands/build.py:51]: Let's see what we use for the "version" when pushing PRs builds from transformers. This assumes there is dev in it, but I was thinking of `pr_xxx` with xxx being the PR number.

Comment (mishig25) [reply src/doc_builder/commands/build.py:51]: sounds good to me

**Comment by mishig25** [comment]
I think this is the only change we need for now to make the dev setup working

File: src/doc_builder/commands/build.py (hunk 1 above)
Comment (sgugger) [reply src/doc_builder/commands/build.py:51]: Ok so it looks like we are going with version being pr_xxx so can you change the dev here to pr?

**Pull request merged**